src/apiserver/db_models.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+import enum
+import time
+import typing
+import pydantic
+
+class RunPhase(enum.Enum):
+    QUEUED = "queued"
+    """
+    Run enters this state when received. 
+    In this state run can be returned to invoker for judging.
+    """
+    LOCKED = "locked"
+    """
+    Some invoker is judging this run right now.
+    There are two possibilities:
+    1) Judging finished, and run transitions to the FINISHED state.
+    2) Timeout is hit, and run transitions to the QUEUED state. (TBD)
+    """
+    FINISHED = "finished"
+    """
+    Judging is finished. All files and statuses are uploaded
+    """
+
+class RunMainProj(pydantic.BaseModel):
+    id: uuid.UUID
+    toolchain_id: str
+    problem_id: str
+    user_id: uuid.UUID
+    contest_id: str
+    phase: str # RunPhase
```

Comment:
Не так важно, но по PEP комменты отделяются двумя пробелами

```suggestion
    phase: str  # RunPhase
```